rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1.java

```diff
@@ -72,4 +72,16 @@
 
 		return new ResponseEntity<>(content, httpHeaders, HttpStatus.OK);
 	}
+
+	@RequestMapping(method = RequestMethod.POST, value = "/file/{fileId}",
+			headers = {"Accept=" + Constants.V1_API_ACCEPT} )
+	public ResponseEntity<String> markFileProcessed(@PathVariable("fileId") String fileId) {
+		API_LOG.info("CPC+ file request received");
```

Comment:
Be a bit more specific with this log entry and the one below.  They're currently the same as in the `getFileById` method.

Reply:
Copy paste gets me again :(